pipelinewise/cli/pipelinewise.py

```diff
@@ -1072,6 +1072,50 @@ def update_state_file_with_extra_log(line: str) -> str:
             with open(tap.state, 'w', encoding='utf-8') as statefile:
                 statefile.write(state)
 
+    def run_tap_partialsync(self, tap: TapParams, target: TargetParams, transform: TransformParams):
+        """Running the tap for partial sync table"""
+
+        # Build the partial sync executable command
+        command = commands.build_fastsync_partial_command(
+            tap=tap,
+            target=target,
+            transform=transform,
+            venv_dir=self.venv_dir,
+            temp_dir=self.get_temp_dir(),
+            table=self.args.table,
+            column=self.args.column,
+            start_value=self.args.start_value,
+            end_value=self.args.end_value
+        )
+
+
+        # Do not run if another instance is already running
+        log_dir = os.path.dirname(self.tap_run_log_file)
+        if (
+                os.path.isdir(log_dir)
+                and len(utils.search_files(log_dir, patterns=['*.log.running'])) > 0
+        ):
```

Comment:
I believe there was a done ticket where we have decided not to use log files as a way to check if tap is running, It think this should also apply to this new command. The existence, or not, of pid file should be enough.

Reply:
- [ ] fix it